File: YazilimGelistiriciDotNetHomeWork/source/repos/KampHomeWorkJune/Day5Homework5GamerBackEnd/Abstracts/BaseGamerManager.cs
using Day5Homework5GamerBackEnd.Concretes;
using System;
using System.Collections.Generic;
using System.Text;

namespace Day5Homework5GamerBackEnd.Abstracts
{
    public abstract class BaseGamerManager : IGamerService
    {
        public virtual void Save(Gamer gamer)
        {
            Console.WriteLine("Saved to database : " + gamer.FirstName);

        }
    }
}

File: JavaReactKampHomeWork/homeworkJavaSE1.8June/src/day4Homework3GamerBackEnd/abstracts/BaseGamerManager.java
package day4Homework3GamerBackEnd.abstracts;

import day4Homework3GamerBackEnd.entities.Gamer;

public abstract class BaseGamerManager implements GamerService{
	@Override
	public void save(Gamer gamer) {
		System.out.println("Saved to database : " + gamer.getFirstName());
	}
	

}
